Tools/LyTestTools/ly_test_tools/__init__.py

```diff
@@ -26,6 +28,7 @@
 HOST_OS_DEDICATED_SERVER = 'unknown'
 HOST_OS_GENERIC_EXECUTABLE = 'unknown'
 HOST_OS_MATERIAL_EDITOR = 'unknown'
+HOST_OS_MATERIAL_CANVAS = 'unknown'
```

Comment:
I'm not 100% sure what these do unless they relate to reporting. Is it necessary to have unique identifiers to distinguish between material editor, material canvas, shader management console, or can they be combined into generic values? Is that what the generic launcher was used for?

Reply:
yes that is what the generic launcher was used for and this is the new pattern
its important we define exactly what LyTestTools will run in this manner in order to avoid any ambiguity

Reply:
but specific in our case would be Atom tools not MaterialEditor/MaterialCanvas/etc. all in their own launchers
so ill create an Atom tools launcher for these tests